smarts/sstudio/types.py

```diff
@@ -511,7 +511,7 @@ class Trip:
     """The type of the vehicle"""
     depart: float = 0
     """Start time in seconds."""
-    actor: TrafficActor = field(init=False)
+    actor: TrafficActor = field(default=None)
```

Comment:
Ah, right this messes with typing it should be
```suggestion
    actor: Optional[TrafficActor] = field(default=None)
```